e2e/api/steps.js

```diff
@@ -366,7 +366,7 @@ const assertValidData = async (data, pageId) => {
   const response = await apiRequest.validatePage(eventName, pageId, caseData);
   const responseBody = await response.json();
 
-  assert.equal(response.status, 200);
+  chai.assert.equal(response.status, 200);
```

Comment:
Let's remove existing `assert` const from this file, so we could import it from chai in the same manner as `expect` is imported (so we could have `assert` instead of `chai.assert` everywhere).